python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/router_pim_sparse_mode.schema.yml

```diff
@@ -21,6 +21,11 @@ keys:
           ssm_range:
             type: str
             description: IPv4 Prefix associated with SSM.
+          register_local_interface:
+            type: str
+            description: |-
+              Local interface to use for PIM register messages.
+              Supported types include Ethernet, Loopback, Management, Port-Channel, Tunnel, Vlan and iif.
```

Comment:
lets remove the supported types as this is not exhaustive

cf what I see on my device:


```
pf1(config-s-gjgj-router-pim-sparse-ipv6)#register local-interface ?
  Dps                  DPS Tunnel Interface
  Ethernet             Ethernet interface
  Loopback             Hardware interface used for looping packets
  Management           Management interface
  Port-Channel         Port-Channel Sub Interface
  Switch               Switch interface
  Tunnel               Tunnel interface
  UnconnectedEthernet  UnconnectedEthernet interface
  Vlan                 Logical interface into a VLAN
  iif                  Use the IIF address of the (S,G) entry
```

Reply:
removed supported types